universal-login-sdk/lib/api/sdk.ts

```diff
@@ -177,6 +190,11 @@ class UniversalLoginSDK {
     return this.blockchainObserver.subscribe(eventType, filter, callback);
   }
 
+  subscribeToBalances(callback: Function) {
+    ensureNotNull(this.balanceObserver, BalanceObserverNotCreated);
```

Comment:
ensure -> fetchBalanceObserver()

Reply:
fetchTokenDetails()

Reply:
> ensure -> fetchBalanceObserver()
But fetchBalanceObserver receives ensName as an argument. Passing ensName to subscribe is kinda scratchy, no?